docs/building-a-form/common-widgets.md

```diff
@@ -0,0 +1,78 @@
+# Common widgets
+
+These common widgets are included in the us-forms-system by default. Set these widgets in a config file while building your form.
+
+Some widgets are associated with particular schema types or formats. There are additional widgets available, but the us-forms-system uses definitions they're included in, rather than just the field. To use a widget, set `ui:widget` for a field to the name of the widget. Widgets are located in [/src/js/widgets](../../src/js/widgets).
+
+Some widgets are passed as a string (e.g., `'ui: widget': 'date'`), while others are passed as the component itself (e.g., `'ui:widget': CurrencyWidget`).
+- Widgets from the [react-jsonschema-form library](./about-the-us-forms-system-library#understanding-react-jsonschema-form-rjsf) include string mappings.
+- Widgets created specifically for US Forms System do not have these mappings, and therefore must be specified by passing the component for the widget directly to the config.
+
+Available widgets are:
+
+- [`ArrayCountWidget`](#arraycountwidget)
+- [`CheckboxWidget`](#checkboxwidget)
+- [`CurrencyWidget`](#currencywidget)
+- [`DateWidget`](#datewidget)
+- [`EmailWidget`](#emailwidget)
+- [`PhoneNumberWidget`](#phonenumberwidget)
+- [`RadioWidget`](#radiowidget)
+- [`SelectWidget`](#selectwidget)
+- [`SSNWidget`](#ssnwidget)
+- [`TextWidget`](#textwidget)
+- [`YesNoWidget`](#yesnowidget)
+
+### `ArrayCountWidget`
```

Comment:
@annekainicUSDS - For each of these, it would be great to have a short, useful description of the widget, similar to those provided for the definitions (see https://github.com/usds/us-forms-system/pull/182/files#diff-15372f86bdceb3c1bb7f70db64ca8471R32). I know some of these are described in "[Available form features...](https://github.com/usds/us-forms-system/blob/master/docs/building-a-form/available-form-features-and-usage-guidelines.md)," but the best wording is tripping me up a bit. Really appreciate your help!

Reply:
Of course! Should I add this to the table or below as regular content?

Reply:
Below as regular content would be excellent, sort of in the fashion of a term/definition pair. The format in the definitions topic seems nice and succinct:

```markdown
### Current or past dates

The common date field with current or past validation set.
```

Reply:
Okay sounds good, I'll create a PR into this one